README.md

```diff
@@ -81,15 +81,35 @@ If you need more detailed instructions, see the [Novice setup guide][novice-setu
 [wiki]: https://github.com/cantino/huginn/wiki
 [novice-setup-guide]: https://github.com/cantino/huginn/wiki/Novice-setup-guide
 
+### Using Vagrant
```

Comment:
I think that should be a subsection of `Develop` to make it clear that it's a development setup and not intended for production usage.